src/test/java/com/cloudbees/jenkins/plugins/bitbucket/Security2033Test.java

```diff
@@ -115,17 +115,6 @@ public void doFillServerUrlItemsSCMSourceWhenUserWithoutPermissionThenReturnEmpt
         }
     }
 
-    @Issue("SECURITY-2033")
-    @Test
-    public void doCheckServerUrlWhenUserWithoutPermissionThenReturnForbiddenStatus() {
-        try (ACLContext aclContext = ACL.as(User.getOrCreateByIdOrFullName(NOT_AUTHORIZED_USER))) {
-            BitbucketSCMSource.DescriptorImpl.doCheckServerUrl(pr, SERVER_URL);
-            fail("Should fail with AccessDeniedException2");
-        } catch (Exception accessDeniedException2) {
-            assertThat(accessDeniedException2.getMessage(), is(NOT_AUTHORIZED_USER + " is missing the Job/Configure permission"));
```

Comment:
Any reason why you're fully deleting this test?

I suppose adapting this error message should be sufficient.

Reply:
Adapted the test